.circleci/config.yml

```
@@ -4,10 +4,10 @@
 #
 version: 2
 jobs:
-  build-rails-5-2:
+  build-rails-6-0:
```

Considering this will change as new versions are released, I think it would be better to rename it like this: 

```suggestion
  build-rails-current
```